yt_dlp/extractor/youtube/_video.py

```diff
@@ -2955,9 +2955,20 @@ def fetch_po_token(self, client='web', context: _PoTokenContext = _PoTokenContex
         # TODO(future): This validation should be moved into pot framework.
         #  Some sort of middleware or validation provider perhaps?
 
+        gvs_bind_to_video_id = False
+        experiments = traverse_obj(ytcfg, (
+            'WEB_PLAYER_CONTEXT_CONFIGS', ..., 'serializedExperimentFlags', {str}, {urllib.parse.parse_qs}))
+        if traverse_obj(experiments, (..., 'html5_generate_content_po_token', 0), get_all=False) == 'true':
```

Comment:
can shorten these lines:
```suggestion
        experiments = traverse_obj(ytcfg, (
            'WEB_PLAYER_CONTEXT_CONFIGS', ..., 'serializedExperimentFlags', {urllib.parse.parse_qs}))
        if traverse_obj(experiments, (..., 'html5_generate_content_po_token', -1, any)) == 'true':
```

&nbsp;

or, I kinda like this better:

```suggestion
        experiments = traverse_obj(ytcfg, (
            'WEB_PLAYER_CONTEXT_CONFIGS', ..., 'serializedExperimentFlags', {urllib.parse.parse_qs}))
        if 'true' in traverse_obj(experiments, (..., 'html5_generate_content_po_token', -1)):
```